packages/client/README.md

```diff
@@ -20,16 +20,20 @@ To install `client` with `npm`/`pnpm`:
 
 1. Configure your package manager to fetch scoped packages from Nexus Registry.
 
-    ```ini
-    # FILE: .npmrc
-    @iroha2:registry=https://nexus.iroha.tech/repository/npm-group/
-    ```
+   ```ini
+   # FILE: .npmrc
+   @iroha2:registry=https://nexus.iroha.tech/repository/npm-group/
+   ```
 
 2. Install the `client` package:
 
-    ```shell
-    npm i @iroha2/client
-    ```
+   ```shell
+   npm i @iroha2/client
+   ```
+
+## Isomorphic transports
+
+Client uses WebSocket and HTTP to communicate with Iroha Peer. Unfortunately, there is no way to do it in an environment-agnostic way. To solve this issue, Client exposes... TODO
```

Comment:
## Isomorphic transports

Iroha Client uses WebSocket and HTTP to communicate with an Iroha Peer. Unfortunately, there is no way to do it in an environment-agnostic way.

To solve this issues, the client provides additional entrypoints for isomorphic adapters, like it is done in [isomorphic-git](https://github.com/isomorphic-git/isomorphic-git/tree/main#getting-started).

### WebSocket

For WebSocket, the client has two entrypoints: `@iroha2/client/web-socket/native` and `@iroha2/client/web-socket/node`.

1. Import adapters:

   Import `@iroha2/client/web-socket/native` for when the native `WebSocket` exists:
   ```
   import { adapter } from '@iroha2/client/web-socket/native'
   ```

   Import `@iroha2/client/web-socket/node` with the `ws` package for Node.js:

   ```
   import { adapter } from '@iroha2/client/web-socket/node'
   ```

2. Create an isomorphic adapter:

   ```
   new Client({ ws: adapter })
   ```

### Fetch

`fetch` could be provided in the same way. However,  `@iroha2/client` does not provide it itself. There are `node-fetch` and `undici` packages that could be used to create an isomorphic `fetch`.

1. Import fetch from `node-fetch` and `undici` packages:

   ```
   import { fetch } from 'undici'
   import fetch from 'node-fetch'
   ```

2. Create an isomorphic fetch:

   ```
   new Client({ fetch })
   ```

Reply:
> 2. Create an isomorphic adapter:
> 2. Create an isomorphic fetch:

I'd say "Create an Iroha Client with isomorphic adapter", or "Inject WebSocket adapter / fetch into Iroha Client"

> fetch could be provided in the same way. However, @iroha2/client does not provide it itself. There are node-fetch and undici packages that could be used to create an isomorphic fetch.

I'd say that `node-fetch` and `undici` **provide** the `fetch` implementation that could be injected into Iroha Client.

Reply:
I'll leave the link to the PR here as well: https://github.com/0x009922/iroha-javascript/pull/1